### PR TITLE
Use Fashion MNIST in example

### PR DESCRIPTION
**Pull request by crcrpar**
#2469

### CODE REVIEW

File: .github/workflows/examples.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name: examples
 on:
   schedule:
     - cron: '0 15 * * *'
+  pull_request: {}
```
Comment (crcrpar) [inline .github/workflows/examples.yml:6]: Commit this change once confirmed.
```suggestion
```

**Comment by crcrpar** [comment]
Finally examples pass: https://github.com/optuna/optuna/actions/runs/678684969

**Review by toshihikoyanase — APPROVED** [review]
The example CI continues failing for about 20 days, but this PR will fix it. 
LGTM. Thank you.

**Comment by toshihikoyanase** [comment]
@not522 Could you review this PR?

**Review by hvy — APPROVED** [review]
LGTM, thanks for the effort and the investigations. I think the sooner the better, so let me merge this PR.

**Pull request merged**